pkg/resolver/searchSchema_test.go

```diff
@@ -26,10 +27,28 @@ func Test_SearchSchema_Query(t *testing.T) {
 	}
 }
 
+func Test_SearchSchema_Query_WithFilters(t *testing.T) {
+	value1 := "openshift"
+	searchInput := &model.SearchInput{Filters: []*model.SearchFilter{{Property: "namespace", Values: []*string{&value1}}}}
+	propTypesMock := map[string]string{"namespace": "string"}
+	// Create a SearchSchemaResolver instance with a mock connection pool.
+	resolver, _ := newMockSearchSchema(t, searchInput, rbac.UserData{CsResources: []rbac.Resource{}}, propTypesMock)
+
+	resolver.userData = rbac.UserData{CsResources: []rbac.Resource{}}
+	sql := `SELECT DISTINCT "prop" FROM (SELECT jsonb_object_keys(jsonb_strip_nulls("data")) AS "prop" FROM "search"."resources" WHERE ("data"->'namespace'?('openshift') AND ("cluster" = ANY ('{}'))) LIMIT 100000) AS "schema"`
+	// Execute function
+	resolver.buildSearchSchemaQuery(context.TODO())
```

Comment:
For tests is better to use `context.Background()`, which generates a new context.
`TODO()` is the same but indicates the implementation is incomplete, so typically won't apply to tests.
```suggestion
	resolver.buildSearchSchemaQuery(context.Background())
```